packages/cutler/lib/commands/print_versions_command.dart

```diff
@@ -14,12 +14,30 @@ class PrintVersionsCommand extends Command<int> {
 
   @override
   int run() {
-    final shorebirdHash = argResults!.rest.first;
+    late final String shorebirdHash;
+    if (argResults!.rest.isEmpty) {
```

Comment:
A named arg with a default value might be cleaner here

Reply:
I hear you.  I was going for more of a `git foo [rev]` feel.